crates/dojo/core/src/world/world_contract.cairo

```diff
@@ -956,6 +963,44 @@ pub mod world {
             }
         }
 
+        /// Panics if a resource is not upgradable.
+        ///
+        /// Upgradable means:
+        /// - the version of the resource must be incremented by 1,
```

Comment:
We don't manage the version anymore, correct?

Reply:
You're right ser ! 👍